app/views/works/index.html.erb

```diff
@@ -0,0 +1,66 @@
+<div class="album-table">
+  <table class="table">
+    <h3>List of Works</h3>
+    <h4>Albums</h4>
+      <tr>
+        <th>Votes</th>
+        <th>Title</th>
+        <th>Created By</th>
+        <th>Published</th>
+        <th>Upvote</th>
+      </tr>
+      <% Work.sort_media('album').each do |album| %>
+        <tr>
+          <td><%= album.votes.count %></td>
+          <td><%= link_to album.title, work_path(album.id) %></td>
+          <td><%= album.creator %></td>
+          <td><%= album.publication_year %></td>
+          <td><%= link_to "Upvote", work_upvote_path(album.id), method: :post, class:"btn btn-primary" %></td>
+      <% end %>
+        </tr>
+  </table>
+</div>
```

Comment:
This code is repeated for each type of work three times almost exactly the same, making it a great candidate for a partial view